### PR TITLE
Update site-configuration-settings.rst

### PR DESCRIPTION
**Pull request by ewwollesen**
Updated note for false setting as it did not appropriately describe the setting.

### CODE REVIEW

File: source/configure/site-configuration-settings.rst [hunk 1]
```diff
@@ -748,7 +748,7 @@ Enable custom user groups
 +---------------------------------------------------------------------------------------------------+-----------------------------------------------------------------------------+
 | - **true**: **(Default)** Users with appropriate permissions can create custom user groups,       | - System Config path: **Site Configuration > Users and Teams**              |
```
Comment (cwarnermm) [inline source/configure/site-configuration-settings.rst:749]:
```suggestion
| - **true**: **(Default)** Users with appropriate permissions can create custom user groups,       | - System Config path: type it here **Site Configuration > Users and Teams**              |
```

File: source/configure/site-configuration-settings.rst [hunk 2]
```diff
@@ -746,9 +746,9 @@ Enable custom user groups
   :start-after: :nosearch:
 
 +---------------------------------------------------------------------------------------------------+-----------------------------------------------------------------------------+
-| - **true**: **(Default)** Users with appropriate permissions can create custom user groups,       | - System Config path: **Site Configuration > Users and Teams**              |
+| - **true**: **(Default)** Users with appropriate permissions can create custom user groups,       | - System Config path: type it here **Site Configuration > Users and Teams**              |
```
Comment (cwarnermm) [inline source/configure/site-configuration-settings.rst:749]:
```suggestion
| - **true**: **(Default)** Users with appropriate permissions can create custom user groups,       | - System Config path: **Site Configuration > Users and Teams**              |
```

File: source/configure/site-configuration-settings.rst [hunk 3]
```diff
@@ -748,7 +748,7 @@ Enable custom user groups
 +---------------------------------------------------------------------------------------------------+-----------------------------------------------------------------------------+
 | - **true**: **(Default)** Users with appropriate permissions can create custom user groups,       | - System Config path: **Site Configuration > Users and Teams**              |
 |   and users can @mention custom user groups in Mattermost conversations.                          | - ``config.json`` setting: N/A                                              |
```
Comment (cwarnermm) [inline source/configure/site-configuration-settings.rst:750]:
```suggestion
|   and users can @mention custom user groups in Mattermost conversations.                          | - ``config.json`` setting: actual setting name                                              |
```

**Review by ewwollesen — COMMENTED** [review]
Added `config.json` setting to the config.

**Review by cwarnermm — APPROVED** [review]
Thanks, @ewwollesen!

**Pull request merged**